docs/cli/configuration.md

```diff
@@ -518,6 +518,25 @@ the `excludedProjectEnvVars` setting in your `settings.json` file.
 - **`CODE_ASSIST_ENDPOINT`**:
   - Specifies the endpoint for the code assist server.
   - This is useful for development and testing.
+- **`GITHUB_PERSONAL_ACCESS_TOKEN`**:
```

Comment:
Because we want to be agnostic to 3P integrations, we don't want to declare these here in our top-level config. Not all users will be using github, and also not all MCP servers/extensions will use the same env var naming. Please remove these